package.json

```diff
@@ -74,9 +81,12 @@
     "express-graphql": "0.6.6",
     "flow-bin": "0.47.0",
     "graphql": "^0.10.1",
+    "husky": "^0.13.4",
```

Comment:
It is, it just got folded right in between dependencies and dev dependencies in the diff view 😛